src/proof.ml

```diff
@@ -0,0 +1,44 @@
+open Core
+open! Lplib
+open Term
+open Timed
+open ConfigBuiltin
+
+(** [is_axim s excluded] return true if [s] ends with string `_ax` and [s] is 
+    different from [excluded]. *)
+let is_axiom : string -> string -> bool = fun s excluded ->
+    if String.length s > 3 && s <> excluded then
+        String.sub s (String.length s - 3) 3 = "_ax"
+    else
+        false
+        
+(** [output file_name sign proof signature package] create a file that contains
+    the deskolemized proof [proof] with the signature [signature]. *)
+let output : string -> Sign.t -> term -> string -> string -> unit = 
+    fun file_name sign proof signature package ->
+    (* get the config from the signature. *)
+    let cfg = get_config sign in
+    (* add [deskolemized] to the file name. *)
+    let file_name = (Filename.remove_extension file_name) ^ "_deskolemized.lp"  in
+    let oc = Format.formatter_of_out_channel (open_out file_name) in
+    Stdlib.(Common.Console.out_fmt := oc);

```

Comment:
perhaps add `let out = Common.Console.out 1 in` before to avoid repeating here many times